StopSelector/js/main.js

```diff
@@ -124,6 +155,41 @@ function stopList(select, stops) {
   });
 }
 
+
+
+/**
+* This function takes in a collection,
+* (array, object, etc) 
+* It iterates over the entire thing, removes
+* duplicate values, and sorts it by the StopID.
+*
+* This isn't really usable for anything other than
+* stops rn. ISSUE: trying to pass an attribute as
+* a string param to the function causes it to
+* throw out everything except one stop.
+* 
+* @param: collection: Object
+* @return: collection: Object <-- the sorted and
+* uniquified version of your input.
+**/
+
+function uniqueifyAndSort(collection){
+  collection = _.uniq(_.union(_.flatten(collection)), _.iteratee('StopId'));
+  collection.sort(function(a,b) {
```

Comment:
If we're going to have [underscore](http://underscorejs.org/#) in the app, might as well use [`_.sortBy`](http://underscorejs.org/#sortBy)